blog/_posts/2021-01-19-scala-native-0.4-release.md

```diff
@@ -0,0 +1,225 @@
+```
+
+We consider changing target triple as a feature for advanced users, and cannot promise it would currently work with any possible configuration yet.
+However, the number of supported architectures and operating systems would definitely grow in the future.
+
+> Upcoming release will no longer support SBT 0.13.x
+
+## Changes to intermediate representation
+
+> NIR (Native Intermediate Representation) is a format used to represent a subset of LLVM instructions, types and values,
+> augmented with a number of high-level primitives that are necessary to efficiently compile Scala,
+> you can read more about in Scala Native [documentation](https://scala-native.readthedocs.io/en/latest/contrib/nir.html).
+
+The new release includes many necessary changes to NIR, starting with the way Strings are stored, 
+through adding new literals like `Val.ClassOf` and last, but not least support for default methods. 
+The last one was introduced to support Scala 2.12 behavior which compiles trait methods as default methods in Java interfaces.
+
+Although we always try to keep our NIR changes to absolute minimum, the number of removed or no longer used primitives used 
+in the previous version forced us to break backward binary compatibility. This decision allows us to remove number of known bugs
+and easier maintenance in the future. NIR format might still change in following minor releases of Scala Native, but they
+would be fully backward-compatible from now on.
+
+## How about existing libraries?
+Scala Native had many [libraries](https://index.scala-lang.org/search?targetTypes=native&scalaNativeVersions=0.4.0-M2) compatible with SN `0.4.0-M2`.
+Unfortunately, the new release is not binary compatible with them, and you'll need to wait until authors will republish them using Scala Native 0.4.0.
+
+However, with increasing interest in running the Scala code natively, it should happen quickly after the release.
+
+## Contributing
+Contributors are always welcome. Tou can support the Scala Native project in multiple ways.
+The most straightforward way of doing so is by working on the plugin itself. 
+Take a look at our [contributors’ guide](https://scala-native.readthedocs.io/en/latest/contrib/index.html).
+
+More importantly however, it is to make our ecosystem more native-compatible. 
+Scala Native compiler does not operate on JVM bytecode, but on its own intermediate representation distributed in `*.nir` files in resulting JAR.
+Library dependencies not compiled with Scala Native would fail when linking. You can help by developing tools
+and libraries not using Java standard library. For example, you can take a look at [sconfig](https://github.com/ekrich/sconfig) - 
+a Scala port of the widely used Lightbend Config library.
+
+## Summary
+Scala Native is starting to keep up with the rest of the Scala ecosystem.
+We expect it will bloom before long with the release, resulting in many libraries and tools.
```

Comment:
```suggestion
We expect it will bloom quickly thanks to this release, resulting in many libraries and tools.
```
expect might be a very strong word here. Maybe *hope* or *believe* or *are confident that*